docs/src/css/custom.css

```
@@ -428,22 +554,103 @@ html[data-theme='dark'] .header--github-link::before {
+}
+
+.change-color:hover {
+  text-decoration: none;
+}
+
+.markdown table,
+.theme-doc-markdown table {
+  display: table;
+  width: 100%;
+  table-layout: fixed;
+  word-wrap: break-word;
```

Stylelint (17.12.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)